server/verify_test.go

```diff
@@ -270,6 +348,54 @@ func TestVerifyBasicAttestation(t *testing.T) {
 	}
 }
 
+func TestVerifyBasicAttestationWithTdx(t *testing.T) {
+	rwc := test.GetTPM(t)
+	defer client.CheckedClose(t, rwc)
+
+	ak, err := client.AttestationKeyRSA(rwc)
+	if err != nil {
+		t.Fatalf("failed to generate AK: %v", err)
+	}
+	defer ak.Close()
+
+	nonce := []byte("super secret nonce")
+	attestation, err := ak.Attest(client.AttestOpts{
```

Comment:
The test says it uses Tdx, but you're not providing a test TDX device.

Reply:
Added TDX device.